imblearn/datasets/zenodo.py

```diff
@@ -0,0 +1,279 @@
+"""Collection of imbalanced datasets.
+
+This collection of datasets have been proposed in [1]_. The
```

Comment:
I think that is: "This collection of datasets **has** been proposed..." or "These datasets have been proposed..."